care-connect/src/components/LandingPage.jsx

```diff
@@ -1,14 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Signup from './Signup'; // Import the Signup component
 
 const LandingPage = () => {
+  const [showSignup, setShowSignup] = useState(false);
```

Comment:
I think this may need to be refactored/removed. We want to show the LandingPage when people hit the base URL for our application. We should be hitting the Signup page only if a user hits the signup button.